addons/mail/static/src/views/fields/statusbar_duration/statusbar_duration_field.js

```diff
@@ -0,0 +1,114 @@
+/** @odoo-module **/
+
+import { _lt } from "@web/core/l10n/translation";
+import {
+    preloadStatusBar,
+    statusBarField,
+    StatusBarField,
+} from "@web/views/fields/statusbar/statusbar_field";
+import { registry } from "@web/core/registry";
+
+export class StatusBarDurationField extends StatusBarField {
+    static template = "mail.StatusBarDurationField";
+
+    humanizeDuration(seconds, showFullDuration) {
```

Comment:
Would also add a docstring, notably for showFullDuration, with a sample input/output